source/appModules/foobar2000.py

```diff
@@ -14,31 +14,38 @@
 # A named tuple for holding the elapsed and total playing times from Foobar2000's status bar
 statusBarTimes = collections.namedtuple('StatusBarTimes', ['elapsed', 'total'])
 
-def getParsingFormat(interval):

```

Comment:
removing this function breaks the add-on API. Please follow steps outlined in `devDocs/deprecations.md` to deprecate the function.

Reply:
alternatively, keep the function